sc/src/test/java/uk/ac/ebi/atlas/search/FeaturedSpeciesDaoIT.java

```diff
@@ -0,0 +1,58 @@
+package uk.ac.ebi.atlas.search;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.support.EncodedResource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.init.ScriptUtils;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.jdbc.JdbcTestUtils;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.inject.Inject;
+
+import java.io.LineNumberReader;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Transactional
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration("classpath:applicationContext.xml")
+class FeaturedSpeciesDaoIT {
+    @Inject
+    private JdbcTemplate jdbcTemplate;
+
+    // @Inject
+    // @Qualifier("dataSource")
+    // private DataSource dataSource;
+
+    private FeaturedSpeciesDao subject;
+
+    @BeforeEach
+    void setUp() {
+        JdbcTestUtils.deleteFromTables(jdbcTemplate, "scxa_experiment");
+        subject = new FeaturedSpeciesDao(jdbcTemplate);
+    }
+
+    @Test
+    void whenNoExperimentsAreLoadedResultsAreEmpty() {
+        assertThat(subject.fetchSpeciesSortedByExperimentCount()).isEmpty();
+    }
+
+    @Test
+    void sortsSpeciesNamesByNumberOfExperimnts() throws Exception {
+        EncodedResource resource = new EncodedResource(new ClassPathResource("uk/ac/ebi/atlas/search/scxa_experiment_fixture.sql"));
```

Comment:
Having read up a little on populating databases in Spring tests, I think there might be less verbose ways of achieving this: 
- Using the `ResourceDatabasePopulator`: https://docs.spring.io/spring/docs/4.3.9.RELEASE/spring-framework-reference/html/integration-testing.html#testcontext-executing-sql-programmatically
- Using `@Sql` annotations: https://docs.spring.io/spring/docs/4.3.9.RELEASE/spring-framework-reference/html/integration-testing.html#testcontext-executing-sql-declaratively

Obviously, your way is also fine if it does the job, but it might be something worth exploring when we implement more robust testing.